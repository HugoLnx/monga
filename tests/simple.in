int main() {
	printf("\n");
	printf("hello \n");
	printf("world\n");
}
