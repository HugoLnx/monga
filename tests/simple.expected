program:
  function:main,259,0
    block:
